platform_db_backup.sh

```diff
@@ -81,6 +87,15 @@ if [[ -v AWS_BACKUP_BUCKET || -v SFTP_SERVER ]]; then
     mkdir -p "${DUMP_FOLDER}"
     echo "Creating database dump to ${DUMP_FILE}";
     time mysqldump --max-allowed-packet=16M --single-transaction --skip-opt -e --quick --skip-disable-keys --skip-add-locks -a --add-drop-table --triggers --routines -h "${DB_HOST}" -P "${DB_PORT}" -u "${DB_USER}" -p"${DB_PASS}" "${DB_NAME}" | gzip -9 > "${DUMP_FILE}"
+    if [ -f ".ssh/secret.key" ]; then
```

Comment:
Would it be usable if we were able to enable/disable encryption w/ env var? As I can see it now, it is always on.

Reply:
good point. I added a variable for it.